libp2p/Cargo.toml

```diff
@@ -12,7 +12,6 @@ categories = ["network-programming", "asynchronous"]
 
 [features]
 full = [
```

Comment:
Removing `async-std` from the top-level `libp2p` crate should be done in a separate PR after it was removed from all other sub-crates.